webpage/src/scripting/methods-and-objects.md

```diff
@@ -870,6 +870,10 @@ script.jumpToNoteSubFolder("a sub folder");
 script.jumpToNoteSubFolder("a sub folder/sub");
 ```
 
+::: tip
+You can create a new subfolder by calling [`mainWindow.createNewNoteSubFolder`](../classes/#example-2)
```

Comment:
Can you please write `new note subfolder in the current subfolder`.
And can you please close the sentence with a `.`. 😁

Reply:
Updated :)

Reply:
Thank you!